### PR TITLE
Updates program length copy

### PR DESCRIPTION
**Pull request by anselmbradford**
## Changes
- Update debt summary text for program durations less than one year.
## Testing
- `gulp build`
- Visit URL with data loaded.
- Select 6 months from estimated years to complete drop-down. 
- Scroll to bottom of page and see the `Loans for half a year (program length)` text in debt summary.
- Change the years to 1 year and to 1.5 years and see the updated summary text in debt summary.
## Review
- @mistergone 
- @marteki 
- @higs4281 
- @amymok 
## Screenshots

screen shot 2016-07-25 at 2 34 08 pm

screen shot 2016-07-25 at 2 31 27 pm
## Notes
- Moves `years` text inside the yearsAttending span tag. I didn't see a problem with this.

### CODE REVIEW

**Comment by higs4281** [comment]
👍 nice!

**Comment by marteki** [comment]
Nice! But the change to the treatment of "years" means that a mild tweak is needed in the content:

screen shot 2016-07-25 at 5 39 29 pm

**Comment by anselmbradford** [comment]
@marteki I see this in the code, but how do I show that text?

```
Currently, your projected monthly student loan
                                  payment is <span id="criteria_loan-monthly"
                                  data-financial="loanMonthly">[$XX]</span>,
                                  based on standard loan terms of <span
                                  data-financial="repaymentTerm"
                                  data-currency="false">[XX]</span> years. Once
                                  you leave school, you are able to choose one
                                  of 
```

Pretty sure the `years` mentioned in there needs to be removed too, but I don't know how to test it.

**Comment by higs4281** [comment]
👍 this fixes @marteki's issue for me.

**Comment by marteki** [comment]
@anselmbradford Good thought to check on the other places we use the word "years", but that one is referring to the length of the loan term. Federal loan terms are usually 10 years, but in some cases can be 25 years. After a search through the template, the only two places we report the program length in years are the summary area that you first fixed, and the paragraph content you just pushed.

**TL;DR** don't worry, that's not the years of program length.You caught them all, so :+1: to merge.
